flask_restful/reqparse.py

```diff
@@ -42,6 +42,8 @@ class Argument(object):
         returned by :meth:`~reqparse.RequestParser.parse_args()`.
     :param bool required: Whether or not the argument may be omitted (optionals
         only).
+    :param bool nullable: Whether or not the argument may be None type (optionals
```

Comment:
I'm confused. You said this was an error with `required=True` but the docs you added say otherwise?